NickvisionMoney.GNOME/Views/MainWindow.cs

```diff
@@ -297,6 +296,11 @@ public MainWindow(MainWindowController controller, Adw.Application application)
         actHelp.OnActivate += Help;
         Handle.AddAction(actHelp);
         application.SetAccelsForAction("win.help", new string[] { "F1" });
+        //Primary Menu Action
+        var actPrimaryMenu = Gio.SimpleAction.New("primaryMenu", null);
+        actPrimaryMenu.OnActivate += PrimaryMenu;
```

Comment:
You can place the callback here in one line:
`actPrimaryMenu.OnActivate += (sender, e) => _btnMainMenu.Popup();`
Same for toggle sidebar

Reply:
Should other cases (Quit, Help) also be changed?

Reply:
yes, I think it's better to change them too